test/e2e_node/restart_test.go

```diff
@@ -84,14 +83,9 @@ var _ = SIGDescribe("Restart [Serial] [Slow] [Disruptive]", func() {
 	f := framework.NewDefaultFramework("restart-test")
 	ginkgo.Context("Container Runtime", func() {
 		ginkgo.Context("Network", func() {
-			ginkgo.It("should recover from ip leak [Flaky]", func() {
+			ginkgo.It("should recover from ip leak", func() {
 				if framework.TestContext.ContainerRuntime == "docker" {
-					bytes, err := ioutil.ReadFile("/etc/os-release")
-					if err != nil {
-						if strings.Contains(string(bytes), "ubuntu") {
-							ginkgo.Skip("Test fails with in-tree docker + ubuntu. Skipping test.")
-						}
-					}
+					ginkgo.Skip("Test fails with in-tree docker + ubuntu. Skipping test.")
```

Comment:
```suggestion
					ginkgo.Skip("Test fails with in-tree docker. Skipping test.")
```
I think this is clearer.